examples/chart/teleport-cluster/values.yaml

```diff
@@ -164,6 +167,19 @@ highAvailability:
     # This defaults to 'cert-manager.io' which is the default Issuer group.
     issuerGroup: cert-manager.io
 
+# Settings for mounting your own TLS keypair to secure Teleport's web UI.
+# These settings are mutually exclusive with the "highAvailability.certManager" and "acme" values above.
+tls:
+  # Name of an existing secret to use which contains a TLS keypair. Will automatically set the https_keypairs section in teleport.yaml.
+  # Create the secret in the same namespace as Teleport using `kubectl create secret tls my-tls-secret --cert=/path/to/cert/file --key=/path/to/key/file`
+  # See https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets for more information.
+  existingSecretName: ""
+  # (optional) Name of an existing secret to use which contains a CA or trust bundle in x509 PEM format.
+  # Useful for building trust when using intermediate certificate authorities.
+  # This will automatically set the SSL_CERT_FILE environment variable to trust the CA.
+  # Create the secret with `kubectl create secret generic --from-file=ca.pem=/path/to/root-ca.pem
+  # The filename inside the secret is important - it _must_ be ca.pem
+  existingSecretNameCA: ""
```

Comment:
nit: should this be named `existingSecretCAName` instead?

Reply:
If that seems clearer I can change it. I think it was `existingCASecretName` in my first iteration which does parse a little better.

Reply:
Changed it to `existingCASecretName` which I think makes more sense.